packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -1165,6 +1165,15 @@ mixin WidgetsBinding on BindingBase, ServicesBinding, SchedulerBinding, GestureB
 /// underlying render tree, similar to what happens when a [StatefulWidget]
 /// rebuilds after calling [State.setState].
 ///
+/// By default, [runApp] will render the provided `app` widget into the
+/// [PlatformDispatcher.implicitView] by wrapping it in a [View] widget, which
+/// will bootstrap the render tree for the app. If an app wants to manually
+/// create and/or manage the [FlutterView](s) into which it is drawn,
+/// `renderIntoDefaultSurface` may be set to false. In that case, the provided
+/// `app` widget tree must contain [View] widget(s) (or widgets that implement
+/// similar functionality) to specify the [FlutterView](s) into which the pixels
+/// for the app are drawn.
+///
```

Comment:
In the future this doc will mention that on desktop platforms runApp will create a window by default into which the provided app is rendered. If developers want to control how that window is created, they can set `renderIntoDefaultSurface` to false and include their own `Window` widget (configured to their liking) in the provided `app.